healthtools_ke_api/build_query.py

```diff
@@ -12,87 +12,76 @@
 
 es = Elastic()
 
+KEYWORDS = {
+        "doctors": ['doc', 'daktari', 'doctor', 'oncologist', 'dr'],
+        "nurses": ['nurse', 'no', 'nursing officer', 'mhuguzi', 'muuguzi', 'RN', 'Registered Nurse'],
+        "clinical-officers": ['co', 'clinical officer', 'clinic officer', 'clinical', 'clinical oficer'],
+        "NHIF": ['nhif', 'bima', 'insurance', 'insurance fund', 'health insurance', 'hospital fund'],
+        "health-facilities": ['hf', 'hospital', 'dispensary', 'clinic', 'hospitali', 'sanatorium', 'health centre']
+        }
 
 class BuildQuery(object):
     def __init__(self):
         self.SMS_RESULT_COUNT = 4  # Number of results to be send via sms
-        self.DOC_KEYWORDS = ['doc', 'daktari', 'doctor', 'oncologist', 'dr']
-        self.CO_KEYWORDS = ['CO', 'clinical officer',
-                            'clinic officer', 'clinical', 'clinical oficer', ]
-        self.NO_KEYWORDS = ['nurse', 'no', 'nursing officer',
-                            'mhuguzi', 'muuguzi', 'RN', 'Registered Nurse']
-        self.NHIF_KEYWORDS = ['nhif', 'bima', 'insurance',
-                              'insurance fund', 'health insurance', 'hospital fund']
-        self.HF_KEYWORDS = ['hf', 'hospital', 'dispensary', 'clinic',
-                            'hospitali', 'sanatorium', 'health centre']
+        
+    def clean_query(self, query): 
+        #removes period from the query, changes the query into a lowercased list '''
```

Comment:
This should be a doc string. Here is an awesome standard to follow https://thomas-cokelaer.info/tutorials/sphinx/docstring_python.html that also makes auto-documentation easier (unless you guys have a different standard you're planning on using)

Reply:
@RyanSept Agreed! We actually use http://docs.python-guide.org/en/latest/writing/documentation/

Reply:
cc @Awinja-Andela